include/zephyr/bluetooth/addr.h

```diff
@@ -81,6 +81,13 @@ static inline int bt_addr_cmp(const bt_addr_t *a, const bt_addr_t *b)
 	return memcmp(a, b, sizeof(*a));
 }
 
+/** @brief Compare Bluetooth LE device addresses for equality.
+ */
```

Comment:
Since this is a public header file you need to have complete doxygen documentation for the input parameters as well as the return value.

Reply:
Is this really a requirement? I don't think there is any information to add here. Structural equality is a generic operation. In a modern language, it would be derived by the compiler automatically and accessible by a generic (==). More text in the documentation does not make for better documentation. I don't even know what to write in the param documentation that would not make me grimace. "bt_addr_le_t *a: The address to compare equal to `b`".? It just seems redundant.

Reply:
How does the generated documentation look like if you omit this info?

Reply:
This is how the doc looks like now:
![image](https://user-images.githubusercontent.com/48479610/196637081-ba1feb44-b8cd-482b-a4e4-35954f203cf7.png)

Reply:
I added a `@sa` to `bt_addr_le_cmp`, which now looks like this:
![image](https://user-images.githubusercontent.com/48479610/196637870-266124a3-8f2c-4896-969a-c80d37764cfb.png)